pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -159,8 +160,9 @@ func (c *Cluster) makeMgrDaemonContainer(mgrConfig *mgrConfig) v1.Container {
 			config.NewFlag("client-mount-gid", "0"),
 			"--foreground",
 		),
-		Image:        c.spec.CephVersion.Image,
-		VolumeMounts: controller.DaemonVolumeMounts(mgrConfig.DataPathMap, mgrConfig.ResourceName),
+		Image:           c.spec.CephVersion.Image,
+		ImagePullPolicy: *controller.GetContainerImagePullPolicy(c.spec.CephVersion.Image, controller.OperatorConfig{}),
```

Comment:
```suggestion
		ImagePullPolicy: *controller.GetContainerImagePullPolicy(c.spec.CephVersion.ImagePullPolicy, controller.OperatorConfig{}),
```
Because of this mgr pod is not coming up